src/sentry/api/endpoints/event_file_committers.py

```diff
@@ -192,6 +193,7 @@ def get(self, _, project, event_id):
         )
 
         committers = self._get_committers(annotated_frames, relevant_commits)
+        metrics.incr('feature.owners.hit' if committers else 'feature.owners.miss')
```

Comment:
This will be more sustainable as `features.hit/features.miss` with an `instance='owners'` kwarg.

Reply:
@JTCunning how so? I dont expect other features to have the same attributes so it seems kind of like a non-optimization from what i recall of datadog usage

Reply:
If we create another feature called `sandwiches`, we'd be using the same metric path and just adding `instance='sandwiches'`. It's just metric path consolidation.

Reply:
why wouldn't you instead say 'feature.owners' instance='hit

Reply:
It's that `hit` and `miss` are at most two separate options with the possibility of N different features. All I'm asking is just a low effort way to reduce metric path clutter.

Reply:
@JTCunning ok just to keep it namespaced then, how about 'feature.owners.has-results' with hits/miss as the instance ('owners' is a much broader bucket, and hit/miss as namespaces dont make a lot of sense)

I dont think it ever makes sense to say 'the metric is feature.hits' because that isn't a generic concept. The outcome of the metric is what the instance should really be (e.g. the response code is an instance, not the type of thing [htt] that generates a response code)

Reply:
Follow your heart.